SimC VS Builds are not uniform

### Describe the bug
When building on VS 2019 we do not copy over a `simc.exe` into the root directory, like we do with vs2017. Instead we just leave it in `x64\Release\simc_vs2019.exe`. 

### Expected behavior
I'd expect both solutions to output a `simc.exe` in the root directory of the project. This is used for various things like the SpellDataDump scripts that expect this to be here.

### To Reproduce
Build the solution with `simc_vs2019.sln`

### Additional Info
I tried fixing this by simply adding `<Import Project="OutputDirectories.props" />` to the build solution, but this breaks the CI build: 

```
Run vs\x64\Debug-NoNetworking\simc_vs2019.exe $env:SIMC_PROFILE iterations=5 output=nul html=nul json2=nul cleanup_threads=1
vs\x64\Debug-NoNetworking\simc_vs2019.exe: D:\a\_temp\5c1f50af-75ab-4b31-9f32-cee96865305e.ps1:2
Line |
   2 |  vs\x64\Debug-NoNetworking\simc_vs2019.exe $env:SIMC_PROFILE iteration …
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The module 'vs' could not be loaded. For more information, run 'Import-Module vs'.

Error: Process completed with exit code 1.
```

This should also rename the project to match, as the 2017 one is just `simc` but the 2019 one is `simc_vs2019.exe`.

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -239,7 +239,7 @@ jobs:
             platform: x64
             configuration: Debug-NoNetworking
             projectFile: vs/simc_vs2019.vcxproj
-            executable: simc.exe
+            executable: vs\simc.exe
             runSmokeTest: true
 
     steps:
@@ -250,10 +250,6 @@ jobs:
       - name: Build
         run: MSBuild.exe /maxcpucount /p:Configuration="${{ matrix.configuration }}" /p:Platform=${{ matrix.platform }} ${{ matrix.projectFile }}
 
-      - name: Mark Solution as Executable
-        if: ${{ matrix.runSmokeTest }}
-        run: chmod +x ${{ matrix.executable }}
-
       - name: Smoke Test
         if: ${{ matrix.runSmokeTest }}
-        run: ./${{ matrix.executable }} $env:SIMC_PROFILE iterations=5 output=nul html=nul json2=nul cleanup_threads=1
+        run: ${{ matrix.executable }} $env:SIMC_PROFILE iterations=5 output=nul html=nul json2=nul cleanup_threads=1

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -239,7 +239,7 @@ jobs:
             platform: x64
             configuration: Debug-NoNetworking
             projectFile: vs/simc_vs2019.vcxproj
-            executable: vs\x64\Debug-NoNetworking\simc_vs2019.exe
+            executable: simc.exe
             runSmokeTest: true
 
     steps:
@@ -250,6 +250,10 @@ jobs:
       - name: Build
         run: MSBuild.exe /maxcpucount /p:Configuration="${{ matrix.configuration }}" /p:Platform=${{ matrix.platform }} ${{ matrix.projectFile }}
 
+      - name: Mark Solution as Executable
+        if: ${{ matrix.runSmokeTest }}
+        run: chmod +x ${{ matrix.executable }}
+
       - name: Smoke Test
         if: ${{ matrix.runSmokeTest }}
-        run: ${{ matrix.executable }} $env:SIMC_PROFILE iterations=5 output=nul html=nul json2=nul cleanup_threads=1
+        run: ./${{ matrix.executable }} $env:SIMC_PROFILE iterations=5 output=nul html=nul json2=nul cleanup_threads=1

--- a/vs/simc_vs2019.vcxproj
+++ b/vs/simc_vs2019.vcxproj
@@ -22,6 +22,7 @@
     <VCProjectVersion>16.0</VCProjectVersion>
     <ProjectGuid>{3347E740-6831-4441-A559-481FD127156F}</ProjectGuid>
     <RootNamespace>simc</RootNamespace>
+    <ProjectName>simc</ProjectName>
     <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
     <PreferredToolArchitecture>x64</PreferredToolArchitecture>
   </PropertyGroup>
@@ -38,6 +39,7 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
   <ImportGroup Label="PropertySheets">
     <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
+    <Import Project="OutputDirectories.props" />
   </ImportGroup>
   <ItemDefinitionGroup>
     <ClCompile>

